frontend/src/v5/ui/components/viewer/cards/card.styles.ts

```diff
@@ -17,7 +17,9 @@
 import styled from 'styled-components';
 import { DashedContainer } from '@controls/dashedContainer/dashedContainer.component';
 
-export const CardContainer = styled.div`
+export const CardContainer = styled.div.attrs({
+	id: 'tickets',
```

Comment:
I don't think this `id` is necessary
(see the comment here [frontend/src/v5/ui/v4Adapter/overrides/cards/tickets.overrides.ts](https://github.com/3drepo/3drepo.io/pull/3685/files/f7ff5fc694db80e3e9de1edcfa01a187460d17c2..75dbf98c34677b75451c009b3db9b40513ee12ae#diff-dabc76402ac0f115b1bf85734bd304fadb1874521e9dbc69e48c498557814a9d))